app-scientific/fricas/autobuild/defines

```diff
@@ -0,0 +1,40 @@
+PKGNAME=fricas
+PKGSEC=math
+
+# Note: x11-lib is used in the HyperDoc.
+PKGDEP__SBCL="gawk x11-lib sbcl"
+PKGDEP__ECL="gawk x11-lib ecl"
+PKGDEP__AMD64="$PKGDEP__SBCL"
+
+# FIXME: Continue out of heap while build
+#PKGDEP__PPC64EL="$PKGDEP__SBCL"
+#PKGDEP__ARM64="$PKGDEP__SBCL"
+PKGDEP="$PKGDEP__ECL"
+
+PKGSUG="texlive emacs"
+
+# Note: In order to build the graphic examples, xvfb
+# must be used, it is in this package.
+#
+# FIXME: Disable now, since fricas2d have bug related to
+# "double free or corruption (!prev)", and I can't fix it:P
+#
+#BUILDDEP="xorg-server"
+
+PKGDES="An advanced computer algebra system"
+
+AUTOTOOLS_AFTER__ECL=(
+    --with-lisp=ecl
+)
+AUTOTOOLS_AFTER__SBCL=(
+    --with-lisp='sbcl --dynamic-space-size 4096'
+    --enable-gmp
+)
+AUTOTOOLS_AFTER__AMD64=("${AUTOTOOLS_AFTER__SBCL[@]}")
+#AUTOTOOLS_AFTER__ARM64="$AUTOTOOLS_AFTER__SBCL"
+#AUTOTOOLS_AFTER__PPC64EL="$AUTOTOOLS_AFTER__SBCL"
+AUTOTOOLS_AFTER=("${AUTOTOOLS_AFTER__ECL[@]}")
+
+# Note: Else sbcl version of the software will not run,
```

Comment:
Else => Lest. Also, since this is SBCL-specific, you should limit it to those architectures:

```
ABSTRIP__AMD64=0
ABSTRIP__PPC64EL=0
```